external/fv3fit/fv3fit/emulation/models/microphysics.py

```diff
@@ -155,3 +155,143 @@ def build(self, data: Mapping[str, tf.Tensor],) -> tf.keras.Model:
                 **self._get_residual_outputs(inputs, data, hidden),
             },
         )
+
+
+@dataclasses.dataclass(frozen=True)
+class Field:
+    output_name: str = ""
+    input_name: str = ""
+    residual: bool = True
+    # only used if residual is True
+    tendency_name: str = ""
+    selection: SliceConfig = dataclasses.field(default_factory=SliceConfig)
+
+
+@dataclasses.dataclass(frozen=True)
+class ZhaoCarrFields:
+    cloud_water: Field = Field(
```

Comment:
It wouldn't work for the current training data, but consider switching these to the new names so the default works going forward.

Reply:
Done.